hedera-node/src/test/java/com/hedera/services/txns/contract/operation/CommonCallSetup.java

```diff
@@ -0,0 +1,39 @@
+package com.hedera.services.txns.contract.operation;
+
+/*
+ * -
+ * ‌
+ * Hedera Services Node
+ * ​
+ * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ *
+ */
+
+import org.hyperledger.besu.datatypes.Address;
+import org.hyperledger.besu.evm.account.Account;
+import org.hyperledger.besu.evm.frame.MessageFrame;
+import org.hyperledger.besu.evm.worldstate.WorldUpdater;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+
+public class CommonCallSetup {
+
+	static void commonSetup(MessageFrame evmMsgFrame, WorldUpdater worldUpdater, Account acc, Address accountAddr) {
```

Comment:
Are you using `accountAddr` anywhere?

Reply:
It's the `MessageFrame`'s first stack item. When ommited, it causes `NullPointerException` in the execution.